src/shared/util.js

```diff
@@ -238,3 +238,22 @@ export function once (fn: Function): Function {
     }
   }
 }
+
+/**
+ * Returns the number of ms corresponding to the text expression.
+ * @param text Time expression with a time unit ('ms' or 's')
+ */
+export function parseTime (text: string): number | typeof NaN {
+  const matched: ?Array<string> = text.match(/(\d+|\d+\.\d+|\.\d+)\s*(s|ms)/i)
+  if (matched) {
+    /* eslint no-unused-vars:0 */
+    const [_, durationText: string, unit: string] = matched
```

Comment:
You don't need explicitly annotate type here since matched must be an array of string. It should be straightforward to any qualified JS developer. Removing it will make code cleaner, I think.